examples/integrations/oci/vault-reactive/src/main/java/io/helidon/examples/integrations/oci/vault/reactive/VaultService.java

```diff
@@ -18,38 +18,57 @@
 
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
-import java.util.Optional;
+import java.util.Date;
 
 import io.helidon.common.Base64Value;
-import io.helidon.common.http.Http;
-import io.helidon.integrations.oci.vault.CreateSecret;
-import io.helidon.integrations.oci.vault.Decrypt;
-import io.helidon.integrations.oci.vault.DeleteSecret;
-import io.helidon.integrations.oci.vault.Encrypt;
-import io.helidon.integrations.oci.vault.GetSecretBundle;
-import io.helidon.integrations.oci.vault.OciVaultRx;
-import io.helidon.integrations.oci.vault.Secret;
-import io.helidon.integrations.oci.vault.Sign;
-import io.helidon.integrations.oci.vault.Verify;
 import io.helidon.webserver.Handler;
 import io.helidon.webserver.Routing;
 import io.helidon.webserver.ServerRequest;
 import io.helidon.webserver.ServerResponse;
 import io.helidon.webserver.Service;
 
+import com.oracle.bmc.keymanagement.KmsCryptoAsyncClient;
+import com.oracle.bmc.keymanagement.model.DecryptDataDetails;
+import com.oracle.bmc.keymanagement.model.EncryptDataDetails;
+import com.oracle.bmc.keymanagement.model.SignDataDetails;
+import com.oracle.bmc.keymanagement.model.VerifyDataDetails;
+import com.oracle.bmc.keymanagement.requests.DecryptRequest;
+import com.oracle.bmc.keymanagement.requests.EncryptRequest;
+import com.oracle.bmc.keymanagement.requests.SignRequest;
+import com.oracle.bmc.keymanagement.requests.VerifyRequest;
+import com.oracle.bmc.secrets.SecretsAsyncClient;
+import com.oracle.bmc.secrets.model.Base64SecretBundleContentDetails;
+import com.oracle.bmc.secrets.model.SecretBundleContentDetails;
+import com.oracle.bmc.secrets.requests.GetSecretBundleRequest;
+import com.oracle.bmc.vault.VaultsAsyncClient;
+import com.oracle.bmc.vault.model.Base64SecretContentDetails;
+import com.oracle.bmc.vault.model.CreateSecretDetails;
+import com.oracle.bmc.vault.model.ScheduleSecretDeletionDetails;
+import com.oracle.bmc.vault.model.SecretContentDetails;
+import com.oracle.bmc.vault.requests.CreateSecretRequest;
+import com.oracle.bmc.vault.requests.ScheduleSecretDeletionRequest;
+
+import static io.helidon.examples.integrations.oci.vault.reactive.OciHandler.ociHandler;
+
 class VaultService implements Service {
-    private final OciVaultRx vault;
+    private final SecretsAsyncClient secrets;
+    private final VaultsAsyncClient vaults;
+    private final KmsCryptoAsyncClient crypto;
     private final String vaultOcid;
     private final String compartmentOcid;
     private final String encryptionKeyOcid;
     private final String signatureKeyOcid;
 
-    VaultService(OciVaultRx vault,
+    VaultService(SecretsAsyncClient secrets,
```

Comment:
You may want to require the [interface](https://docs.oracle.com/en-us/iaas/tools/java/2.21.0/com/oracle/bmc/secrets/SecretsAsync.html) instead

Reply:
Fixed.